.github/workflows/auto-build.yml

```diff
@@ -19,8 +19,8 @@ jobs:
           echo $version
           arch=`echo "$commentbody"|cut -d" " -f3`
           echo $arch
-          wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/rootfs.tar.gz
-          tar -xvf rootfs.tar.gz
+          wget https://sealer.oss-cn-beijing.aliyuncs.com/auto-build/rootfs-07.tar.gz
```

Comment:
Sealer version as an argument too. https://sealer.oss-cn-beijing.aliyuncs.com/sealers/sealer-${sealerversion}-linux-amd64.tar.gz

Reply:
Substituting sealer-latest achieved the same effect